productService/serverless.yml

```diff
@@ -0,0 +1,127 @@
+# Welcome to Serverless!
```

Comment:
[Recommendations]

1) Upgrade Node to 18 version (currently in this file 12)
2) Use Typescript
3) Use `ts` instead of `yml` -> you can take a look at serverless `aws-nodejs-typescript` template